ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -237,19 +234,6 @@ module.exports = Marionette.LayoutView.extend({
     )
     this.handleCurrentQuery()
 
-    // listener for when the measurement state changes in map model
-    this.listenTo(
```

Comment:
why are these listeners being removed?

Reply:
i think this is actually an issue with the cherry-picking, it seems that my method of doing this overwrites changes in ddf-ui. I'll tag this pr with WIP for now so i can address these